client/src/views/Simulation/Simulation.vue

```diff
@@ -1,74 +1,53 @@
 <template>
   <div class="view-container">
-    <div class="d-flex flex-column flex-grow-1 position-relative">
-      <div class="search-row">
-        <div class="search-col flex-column">
-          <search-bar />
-        </div>
-        <div class="search-col mx-3 justify-content-between">
-          <button class="btn btn-primary blue m-1">
-            Simulate
-          </button>
-          <button class="btn btn-primary m-1">
-            <font-awesome-icon :icon="['fas', 'project-diagram' ]" />
-            <span>Provenance Graph </span>
-          </button>
-          <button class="btn btn-primary m-1" @click="fetchParameters">
-            <span>Reset</span>
-          </button>
-          <button class="btn btn-primary m-1" @click="incrParametersMaxCount">
-            <span>Save</span>
-          </button>
-          <!-- <div class="btn-group m-1" role="group" aria-label="Basic example">
-            <button type="button" class="btn btn-primary" @click="onCloseSimView">Reset</button>
-            <button type="button" class="btn btn-primary" @click="onCloseSimView">Save</button>
-          </div> -->
-        </div>
-        <div class="search-col justify-content-end">
-          <div class="form-check form-check-inline">
-            <input class="form-check-input" type="checkbox" id="inlineCheckbox1" value="option1">
-            <label class="form-check-label" for="inlineCheckbox1">Auto Run</label>
-          </div>
-          <button class="btn btn-primary blue m-1" @click="onCloseSimView">
-            <font-awesome-icon :icon="['fas', 'play' ]" />
-            <span>Run</span>
-          </button>
-          <button class="btn btn-primary m-1" @click="onCloseSimView">
-            <font-awesome-icon :icon="['fas', 'chart-line' ]" />
-            <span> Close Simulation View </span>
-          </button>
-        </div>
+    <div class="search-row">
+      <div class="search-col flex-column">
+        <search-bar />
+      </div>
+      <div class="search-col mx-3 justify-content-between">
+        <button class="btn btn-primary blue m-1">
+          Simulate
+        </button>
+        <button class="btn btn-primary m-1">
+          <font-awesome-icon :icon="['fas', 'project-diagram' ]" />
+          <span>Provenance Graph </span>
+        </button>
+        <button class="btn btn-primary m-1" @click="fetchParameters">
+          <span>Reset</span>
+        </button>
+        <button class="btn btn-primary m-1" @click="incrParametersMaxCount">
+          <span>Save</span>
```

Comment:
These 2 buttons have to do with saving the experiment and resetting the experiments to the default configuration. I think we should move them closer to the `Run` button and maybe we can replace them with just icons to save some space? Since they are not as important we can justify that decision.